frontend/integration-tests/protractor.conf.ts

```diff
@@ -20,6 +21,8 @@ const browserLogs: logging.Entry[] = [];
 
 const suite = (tests: string[]) => (!_.isNil(process.env.BRIDGE_KUBEADMIN_PASSWORD) ? ['tests/login.scenario.ts'] : []).concat(['tests/base.scenario.ts', ...tests]);
 
+const pluginSuites = _.mapValues(pluginsIntegrationTests(), (tests:string[]) => suite(tests));
```

Comment:
@yaacov I just realized, what if a test suite of some plugin needs to have a test spec that has to run before `base.scenario.ts`? In kubevirt case, it is the `kubevirt.login.scenario.ts` that logs in and is a prerequisite for the `base.scenario.ts` to create a test namespace. So login needs to happen before the `base.scenario.ts`.
For that reason I think we cannot the `suite()` wrapper and plugin specific suite will need to specify the path to this file in their suite.

Reply:
:+1: Thanks !

Removed the automatic `suite()` and added an option to call explicitly call `@console/integration-tests/`